website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md

```diff
@@ -128,15 +128,14 @@ if __name__ == '__main__':
 
 4. Replace **`<YOUR_BASE_URL>`** and **`<YOUR_ACCOUNT_ID>`** with the correct values of your environment and [Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
 
-:::tip
- To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://cloud.getdbt.com/deploy/000000/projects/111111/jobs/222222`
-and therefore valid code would be:
+    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://cloud.getdbt.com/deploy/000000/projects/111111/jobs/222222`
```

Comment:
i think a tip callout is helpful in this context bc it's a helpful 'fyi' and not a requirement step. also it's a lot to scan or read. the code under 'Your URL is structured...' needs code blocks too. here's how it looks in the staging and how it currently looks:

![Screenshot 2024-01-09 at 11 06 23](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/c1d8b39c-c5b6-4613-b9e0-b416675ab309)
![Screenshot 2024-01-09 at 11 06 35](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/35ab2f70-9964-49d1-a1c3-aa37892da3a1)

Reply:
I was trying to keep callouts brief and avoid them containing code. Let's see what the group says in the meeting